TELCODOCS-2171#Generalize Day2Ops Observability (#96104)
openshift/openshift-docs: modules/observability-monitoring-the-edge.adoc

// Module included in the following assemblies:
//
// * edge_computing/day_2_core_cnf_clusters/observability/observability.adoc

:_mod-docs-content-type: PROCEDURE
[id="observability-monitoring-the-edge_{context}"]

= Monitoring the edge

OpenShift clusters at the edge should keep the footprint of the platform components to a minimum. 

> The following procedure is an example of how you can configure a {sno} or edge node with a small monitoring footprint.

Reviewer: Same comment as above, in this section, you could refer to "nodes at the far edge of the network", rather than just "the edge" or "edge node"

> The following procedure is an example of how you can configure a {sno} or a node at the far edge network with a small monitoring footprint.

Reviewer 2: ```suggestion
The following procedure is an example of how to configure a {sno} or a node at the far edge network with a small monitoring footprint.
```

  .dockerconfigjson: 'PULL_SECRET'
----

. Apply the `Secret` CR by running the following command in the hub cluster:
+
[source,terminal]
----
$ oc apply -f monitoringSecret.yaml
----

> . Get the keys for the NooBaa service and the back-end bucket name from the hub cluster by running the following commands:

Reviewer: Should NooBaa be wrapped in backticks?
